src/chainparams.cpp

```diff
@@ -90,11 +86,17 @@ class CMainParams : public CChainParams {
         vSeeds.push_back(CDNSSeedData("xf2.org", "bitseed.xf2.org")); // Jeff Garzik
         vSeeds.push_back(CDNSSeedData("bitcoin.jonasschnelli.ch", "seed.bitcoin.jonasschnelli.ch")); // Jonas Schnelli
 
-        base58Prefixes[PUBKEY_ADDRESS] = std::vector<unsigned char>(1,0);
-        base58Prefixes[SCRIPT_ADDRESS] = std::vector<unsigned char>(1,5);
-        base58Prefixes[SECRET_KEY] =     std::vector<unsigned char>(1,128);
-        base58Prefixes[EXT_PUBLIC_KEY] = boost::assign::list_of(0x04)(0x88)(0xB2)(0x1E).convert_to_container<std::vector<unsigned char> >();
-        base58Prefixes[EXT_SECRET_KEY] = boost::assign::list_of(0x04)(0x88)(0xAD)(0xE4).convert_to_container<std::vector<unsigned char> >();
+        const unsigned char chPubkeyAddress[] = { 0 };
+        const unsigned char chScriptAddress[] = { 5 };
+        const unsigned char chSecretKey[] =     { 128 };
+        const unsigned char chExtPublicKey[] =  { 0x04, 0x88, 0xB2, 0x1E };
+        const unsigned char chExtSecretKey[] =  { 0x04, 0x88, 0xAD, 0xE4 };
```

Comment:
Well in fairness, the last two steps reduced the boost dependency, this just goes the final step and actually removes it. I agree that the gradual touching and "improving" can get annoying, but I don't think there's been any flip-flopping here, just small steps.

I understand if you'd rather hold on this and use initializer lists instead, but I may pick up this fight again if this would be usable in a lib before we move to c++11 :)

In fact, the last one wouldn't